### ISSUE TITLE
bug: No difference in colours between selected and unselected button

### ISSUE THREAD

**Issue by robertmonka** [EdenEast/nightfox.nvim#382]
### Neovim version (nvim -v)

0.10

### Operating system/version

Windows 10 + Windows Terminal

### Describe the bug

nightfox.lua + lazy.lua in nvim

In Lazy operating window (:Lazy) I can not see difference between Not selected button and selected button. They both have the same colour. For example Ctrl + I for Install button.

There is no difference between "cursor line" and "text section", also here the same colour

### Steps To Reproduce

Install nvim
Install Lazy
Install colorscheme
Set noxhtfox colorscheme

### Expected Behavior

Change colour of the seclection to more brightnes

### Repro

```Lua
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "https://github.com/folke/lazy.nvim.git",
    "--branch=stable", -- latest stable release
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)

require("lazy").setup({ import = "robert.plugins" }, {
  install = {
    colorscheme = { "nightfox" },
  },
  checker = {
    enabled = true,
    notify = false,
  },
  change_detection = {
    notify = false,
  },
})
```

**Comment by robertmonka** [comment]
![image](https://github.com/EdenEast/nightfox.nvim/assets/88946049/990d2f8f-db31-4f17-885d-eb06457ff3ed)

Clean option selected

**Comment by EdenEast** [comment]
Resolved in #383.
